packages/frontend/src/view/components/LayerZeroMultisig.tsx

```diff
@@ -6,29 +6,45 @@ interface LzMultisigProps {
   owners?: EthereumAddress[]
 }
 
-export function LzMultisig(props: LzMultisigProps): JSX.Element {
+export function LzMultisig({
+  owners,
+  address,
+  threshold,
+}: LzMultisigProps): JSX.Element {
+  // Currently lack of multisig data is dictated either lack of support for multisig for given chain or we lack some data (this must be addressed in the future)
+  const hasData = address && threshold && owners
+
   return (
     <section className="mx-6 mb-12 border-t border-blue bg-gray-900">
       <div className="flex items-center justify-between p-8">
         <h2 className="text-2xl text-lg font-medium text-blue">
           LayerZero Multisig
         </h2>
-        <span className="font-mono text-gray-600">{props.address}</span>
-      </div>
-      <div className="flex border-y border-black bg-gray-800 px-8 py-3">
-        <span className="w-[214px] font-medium text-gray-500">Threshold</span>
-        <span className="font-mono">
-          {props.threshold &&
-            props.owners &&
-            `${props.threshold}/${props.owners.length}`}
-        </span>
-      </div>
-      <div className="flex items-center bg-gray-800 px-8">
-        <span className="w-[214px] font-medium text-gray-500">Owners</span>
-        <pre className="grow overflow-auto border border-gray-700 bg-gray-900 p-6 font-mono text-gray-500">
-          {JSON.stringify(props.owners, null, 2)}
-        </pre>
+        {address ? (
+          <span className="font-mono text-gray-600">{address}</span>
+        ) : (
+          <span className="font-mono text-gray-600">
+            This chain does not support LayerZero Multisig
+          </span>
+        )}
```

Comment:
I'm proposing 2 solutions when the LZ Multisig is not present: 
1. (preferred) - Change the name to "Contracts Owner" and mark it as <problematic (design to be defined)>. We can say that GnosisSafe is not supported on this chain, but only if we're sure about it (Gnosis API?)
2. Just do not display this component, we will mark it <problematic> next to the `owner` contract value field both on ULNv2 and Endpoint

Reply:
I hope you have nothing against doing that as a separate task and merging this one as-is. There is much more going behind the scenes, and I would end-up creating temporary fix ups either way. Let's discuss best approach here async and do the actual implementation separately